pystiche_papers/ulyanov_et_al_2016/_nst.py

```diff
@@ -152,9 +154,9 @@ def stylization(
         transformer = _transformer(
             style=style, impl_params=impl_params, instance_norm=instance_norm,
         )
-        if instance_norm or not impl_params:
-            # https://github.com/pmeier/texture_nets/blob/aad2cc6f8a998fedc77b64bdcfe1e2884aa0fb3e/test.lua#L32
-            transformer = transformer.eval()
+    if instance_norm or not impl_params:
```

Comment:
I think we should change the conditional to make it more clear:

```suggestion
    if not (impl_params and not instance_norm):
```

Although they are the same the suggested version makes it clear that there is only a single case where we don't switch to evaluation mode.
Thoughts?